app/components/Views/ImportFromSecretRecoveryPhrase/index.js

```diff
@@ -270,342 +557,427 @@ const ImportFromSecretRecoveryPhrase = ({
+                  >
                     {strings(
-                      `choose_password.${secureTextEntry ? 'show' : 'hide'}`,
+                      'import_from_seed.enter_your_secret_recovery_phrase',
                     )}
                   </Text>
-                </TouchableOpacity>
+                  <TouchableOpacity onPress={showWhatIsSeedPhrase}>
+                    <Icon
+                      name={IconName.Info}
+                      size={IconSize.Md}
+                      color={colors.icon.alternative}
+                    />
+                  </TouchableOpacity>
+                </View>
+                <View style={styles.seedPhraseRoot}>
+                  <View style={styles.seedPhraseContainer}>
+                    <View style={styles.seedPhraseInnerContainer}>
+                      {seedPhrase.length <= 1 ? (
+                        <TextInput
+                          ref={(ref) => {
+                            seedPhraseInputRefs.current[0] = ref;
+                          }}
+                          textAlignVertical="top"
+                          label={strings('import_from_seed.srp')}
+                          placeholder={strings(
+                            'import_from_seed.srp_placeholder',
+                          )}
+                          value={seedPhrase?.[0] || ''}
+                          onChangeText={(text) =>
+                            handleSeedPhraseChange(text, 0)
+                          }
+                          style={styles.seedPhraseDefaultInput}
+                          placeholderTextColor={colors.text.alternative}
+                          placeholderStyle={
+                            styles.seedPhraseDefaultInputPlaceholder
+                          }
+                          multiline
+                          autoFocus
+                          onKeyPress={(e) => handleKeyPress(e, 0)}
+                          autoComplete="off"
+                          blurOnSubmit={false}
+                          autoCapitalize="none"
+                          testID={
+                            ImportFromSeedSelectorsIDs.SEED_PHRASE_INPUT_ID
+                          }
+                        />
+                      ) : (
+                        <View
+                          style={[styles.seedPhraseInputContainer]}
+                          onLayout={handleLayout}
+                        >
+                          <FlatList
+                            data={seedPhrase}
+                            numColumns={numColumns}
+                            keyExtractor={(_, index) => index.toString()}
+                            renderItem={({ item, index }) => (
+                              <View
+                                style={[
+                                  {
+                                    width: containerWidth / 3,
+                                    padding: inputPadding,
```

Comment:
InputPadding variable is only used here, should we just use the code?
```suggestion
                                    padding: Platform.OS === 'ios' ? 4 : 3,
```